tests/test_fetch.py

```diff
@@ -108,7 +109,7 @@ def test_get_pypi_plugins():
     [
         "https://files.pythonhosted.org/packages/fb/01/e59bc1d6ac96f84ce9d7a46cc5422250e047958ead6c5693ed386cf94003/napari_dv-0.3.0.tar.gz",  # noqa
         "https://files.pythonhosted.org/packages/5d/ae/17779e12ce60d8329306963e1a8dec608465caee582440011ff0c1310715/example_plugin-0.0.7-py3-none-any.whl",  # noqa
-        "git+https://github.com/DragaDoncila/example-plugin.git",
+        # "git+https://github.com/DragaDoncila/example-plugin.git", Draga hide package
```

Comment:
The code coverage is dropping because this is the only example tested that is fetching using git/git+https. We can cover by replacing this with one of the other examples [here](https://app.codecov.io/gh/napari/npe2/pull/319/blob/src/npe2/_inspection/_fetch.py#L231) e.g. git+https://github.com/tlambert03/napari-dv.git